rubygems-update.gemspec

```diff
@@ -6,14 +6,14 @@ Gem::Specification.new do |s|
   s.authors = ["Jim Weirich", "Chad Fowler", "Eric Hodel", "Luis Lavena", "Aaron Patterson", "Samuel Giddins", "André Arko", "Evan Phoenix", "Hiroshi SHIBATA"]
   s.email = ["", "", "drbrain@segment7.net", "luislavena@gmail.com", "aaron@tenderlovemaking.com", "segiddins@segiddins.me", "andre@arko.net", "evan@phx.io", "hsbt@ruby-lang.org"]
 
-  s.summary = "RubyGems is a package management framework for Ruby."
+  s.summary = "RubyGems is a package management framework for Ruby—this gem allows the gem command to update itself."
   s.description = "A package (also known as a library) contains a set of functionality
   that can be invoked by a Ruby program, such as reading and parsing an XML file. We call
   these packages 'gems' and RubyGems is a tool to install, create, manage and load these
   packages in your Ruby environment. RubyGems is also a client for RubyGems.org, a public
   repository of Gems that allows you to publish a Gem that can be shared and used by other
   developers. See our guide on publishing a Gem at guides.rubygems.org"
-  s.homepage = "https://rubygems.org"
+  s.homepage = "https://github.com/rubygems/rubygems/"
```

Comment:
I think I agree with this change 👍.

Reply:
I think we can point to https://guides.rubygems.org/ in here.